bdd-generator/src/main/java/com/epam/test_generator/services/StepSuggestionService.java

```diff
@@ -140,6 +141,7 @@ public StepSuggestionDTO updateStepSuggestion(Long projectId,
         checkNotNull(stepSuggestion);
         stepSuggestion.verifyVersion(stepSuggestionUpdateDTO.getVersion());
         stepSuggestionTransformer.updateFromDto(stepSuggestion, stepSuggestionUpdateDTO);
+        stepSuggestion.setLastUsedDate(ZonedDateTime.now());
```

Comment:
Method updateFromDto does not look weel in transformer, it's more like updater. Update the logic per our 1-1 discussion.

Reply:
I agree, I can move this method in service, but we have these methods in all transformers. Should I move all methods in services or it's better to leave everything as it was?

Reply:
leave as is